crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -199,6 +199,34 @@ impl CliOutput {
             }
         }
     }
+
+    fn print_failed_tasks(&self, detail: &JobDetail, worker_map: &WorkerMap) {
```

Comment:
This is basically the same thing as `print_task_summary`. These functions should do the same thing - display a summary of failed tasks. Could you try to unify them please?